src/map/battle.cpp

```diff
@@ -4248,6 +4248,9 @@ static int battle_calc_attack_skill_ratio(struct Damage* wd, struct block_list *
 			skillratio += -100 + 400 * skill_lv;
 			RE_LVL_DMOD(100);
 			break;
+		case NPC_IGNITIONBREAK:
+			skillratio += -100 + 1000 * skill_lv;	// Unknown after job improvement
```

Comment:
NPC_* skill usually never change when there are changes to the player based ones. So as long as the formula being used is the one prior to the skill balancing patches.